demo/demo_director.py

```diff
@@ -383,10 +380,11 @@ def revoke_compromised_keys():
     repository.snapshot.add_verification_key(new_snapshot_public_key)
 
     # Unload the old signing keys so that the new metadata only contains
-    # signatures produced by the new signing keys.
-    repository.targets.unload_signing_key(old_targets_private_key)
-    repository.snapshot.unload_signing_key(old_snapshot_private_key)
-    repository.timestamp.unload_signing_key(old_timestamp_private_key)
+    # signatures produced by the new signing keys. Since this is based on
+    # keyid, the public key can be used.
+    repository.targets.unload_signing_key(old_targets_public_key)
```

Comment:
It looks a little odd to use the public keys with unload_signing_key(), but it does reduce clutter.